libextract/quantifiers.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+"""
+A submodule providing 'quantifying' functions.
+
+The protocol that's followed is each function expects a "node"
+of type lxml.html.HtmlElement (sometimes *._ElementTree), and
+the function will return either a numerical type or a Counter
+where the key=<node> and value=<numerical>
+"""
+
+from libextract.coretools import Counter
```

Comment:
We don't explicitly export Counter, better to import it from the collections module.